app/konnect/service-catalog/integrations/traceable.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Traceable Integration
+content-type: reference
+config:
+    discovery_support: true
+    discovery_default: true
+    bindable_entities: "Traceable Service"
+    mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The Traceable integration lets you connect Traceable entities directly to your Service Catalog services.
+
+## Authenticate the Traceable integration
+
+1. From {{site.konnect_product_name}} select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
+2. Select **Traceable**, then **Install Traceable**.
+3. Select **Authorize**. 
+
+## Bindable entities
+
+Entity | Binding Level | Description
+-------|---------------|-------------
+Traceable Service | Service | [Traceable Service](https://docs.traceable.ai/docs/domains-services-backends) which holds group of Traceable API endpoint resource
+
+## Discovery information
+
+{% include_cached service-catalog-discovery.html %}
```

Comment:
This also incorrectly says "Repositories".

<img width="2032" alt="Screenshot 2024-09-10 at 4 29 41 PM" src="https://github.com/user-attachments/assets/bdc6ed2b-57db-413a-bf57-621448466315">